src/elements/content-uploader/__tests__/ContentUploader.test.js

```diff
@@ -119,6 +119,40 @@ describe('elements/content-uploader/ContentUploader', () => {
             const expected = { yoyo: true, yoyo_0_10000: true };
             expect(wrapper.state().itemIds).toEqual(expected);
         });
+
+        test.only('should handle accepting package "files" separate from folders', () => {
```

Comment:
I think you don't want `test.only()` here right?

Reply:
good catch!